### ISSUE TITLE
.NET Analyzers

### ISSUE THREAD

**Issue by mzorec** [ironcev/awesome-roslyn#12]
There is an organization [.NET Analyzers.](https://github.com/DotNetAnalyzers) There are some useful analyzers in this organization. I've opened this issue because I am not sure if the organization should be added or individual repositories in the organization?

StyleCopAnalyzers repository from this organization is already added on awesome-roslyn.

**Comment by ironcev** [comment]
Hi @mzorec,

@DotNetAnalyzers are known to me and I am amazed by the number and the quality of analyzers available in various repositories within the organization. I asked myself the same question, should it be added as an organization, or the individual repositories should be added. I had issues with both approaches.

At first, I wanted to add the organization because it surely deserves to be added. Adding the organization clashed a bit with the existing approach where only individual projects/repositories are on the list. Also, and this is a bit bigger issue for me, not all of the individual repositories fulfill the [critieria listed in the contribution guidelines](https://github.com/ironcev/awesome-roslyn/blob/master/contributing.md#open-source-projects).

When I thought of adding more individual repositories, the fact that some of them do not fulfill the basic criteria (e.g. having a proper README page) was an issue. I thought of contributing to those projects to help them reach the basic critieria, but simply didn't find time. Also, there are many fine-grained repositories within the organization. Even if they fulfilled the criteria, I doubt it would make sense to add them all as separate projects.

Long story short, please give me some time to (re)think :-) @DotNetAnalyzers definitely deserve to be added to the list, it's just the question how.

**Comment by ironcev** [comment]
Hi @mzorec, thanks for your patience. It took me a while to come back to this proposal. I've finally added @DotNetAnalyzers as an organization and emphasized that in the description part.
